ZFSin/zfs/include/sys/zfs_ioctl.h

```diff
@@ -598,8 +598,9 @@ typedef enum zfs_ioc {
 	//ZFS_IOC_WINDOWS= ('Z' << 8) + 0xE0,
 	ZFS_IOC_MOUNT				= CTL_CODE(ZFSIOCTL_TYPE, 0x8E0, METHOD_NEITHER, FILE_ANY_ACCESS),
 	ZFS_IOC_UNMOUNT				= CTL_CODE(ZFSIOCTL_TYPE, 0x8E1, METHOD_NEITHER, FILE_ANY_ACCESS),
+	ZFS_IOC_UNREGISTER_FS		= CTL_CODE(ZFSIOCTL_TYPE, 0x8E2, METHOD_NEITHER, FILE_ANY_ACCESS),
 
-	ZFS_IOC_LAST				= CTL_CODE(ZFSIOCTL_TYPE, 0x8E2, METHOD_NEITHER, FILE_ANY_ACCESS),
+	ZFS_IOC_LAST				= CTL_CODE(ZFSIOCTL_TYPE, 0x8E3, METHOD_NEITHER, FILE_ANY_ACCESS),
 } zfs_ioc_t;
 
```

Comment:
Technically, the IOCTLs actually live in `include/sys/fs/zfs.h` on illumos, I had to move them into `zfs_ioctl.h` for OSX. So if we want to break them out into its own file and just include, that is fine.

Reply:
ok, thats good. More work for times when everything else is overly frustrating again :smile: